src/python/ensembl/io/genomio/genbank/get_genbank.py

```diff
@@ -33,48 +34,47 @@ def __init__(self, msg):
         self.msg = msg
 
 
-class InputSchema(argschema.ArgSchema):
-    """Input arguments expected by this script."""
-
-    accession = argschema.fields.String(
-        metadata={"required": True, "description": "Sequence accession required"}
-    )
-
-
-def download_genbank(accession: str) -> str:
+def download_genbank(accession: str, output_gb: PathLike) -> None:
     """
-    Given a GenBank accession, download via NCBI Enterez service the corresponding file in GenBank format.
+    Given a GenBank accession, download via NCBI Entrez service the corresponding file in GenBank format.
 
     Args:
-        accession: Only param required. Associated with genome INSDC Genbank record.
-
-    Returns:
-        Single file containing a genome sequence record in Genbank format (.gb).
+        accession: INSDC Genbank record accession.
+        output_gb: Path to the downloaded record in Genbank format.
     """
-    dl_file = f"{accession}.gb"
 
     # Get the list of assemblies for this accession
-    e_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi"
-    e_params = {
+    entrez_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi"
+    entrez_params = {
         "db": "nuccore",
         "rettype": "gbwithparts",
         "retmode": "text",
     }
-    e_params["id"] = accession
-    result = requests.get(e_url, params=e_params, timeout=60)
+    entrez_params["id"] = accession
+    result = requests.get(entrez_url, params=entrez_params, timeout=60)
     if result and result.status_code == 200:
-        with open(dl_file, "wb") as gbff:
+        with Path(output_gb).open("wb") as gbff:
             gbff.write(result.content)
-        print(f"GBF file write to {dl_file}")
-        return dl_file
+        print(f"GBF file write to {output_gb}")
+        return
     raise DownloadError(f"Could not download the genbank ({accession}) file: {result}")
 
 
+class InputSchema(argschema.ArgSchema):
+    """Input arguments expected by this script."""
+
+    accession = argschema.fields.String(
+        metadata={"required": True, "description": "Sequence accession required"}
+    )
+    output_gb = argschema.fields.OutputFile(metadata={"required": True, "description": "Ouput Genbank path"})
```

Comment:
```suggestion
    output_file = argschema.fields.OutputFile(metadata={"required": True, "description": "Output GBF file path"})
```
I find this kind of variable naming makes the code easier to maintain in the long term (format/source may change, but it will still produce a file).

Reply:
will do